api/v1alpha1/storageconsumer_types.go

```diff
@@ -100,6 +129,18 @@ type ClientStatus struct {
 	ID string `json:"clientId,omitempty"`
 }
 
+type RadosNamespaceSpec struct {
+	Name string `json:"name"`
+}
+
+type SubVolumeGroupSpec struct {
+	Name string `json:"name"`
+}
+
+type ClientProfileSpec struct {
+	Name string `json:"name"`
+}
+
```

Comment:
I think you meant to have these with a Status safix

Reply:
yes, done.